### PR TITLE
Cleans up some text formatting

### PR DESCRIPTION
**Pull request by Cheridan**
partial revert of #16376, https://github.com/tgstation/-tg-station/issues/16353#issuecomment-201366032 was correct, bold is reserved for userdanger. The sound effect should be warning enough.

### CODE REVIEW

**Comment by Cheridan** [comment]
>Cheridan added some commits on Nov 13, 2014 

magnets

**Comment by KorPhaeron** [comment]
A PR years in the making

**Comment by optimumtact** [comment]
Now we know what cheridan has been doing in all this time

File: code/game/gamemodes/miniantags/revenant/revenant_abilities.dm [hunk 1]
```diff
@@ -114,8 +114,8 @@
 				charge_counter = charge_max
 				return
 			log_say("RevenantTransmit: [key_name(user)]->[key_name(M)] : [msg]")
-			user << "<span class='revennotice'><b>You transmit to [M]:</b> [msg]</span>"
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/revenant/revenant_abilities.dm]: this was intentional so the message wasn't bold

File: code/game/gamemodes/miniantags/revenant/revenant_abilities.dm [hunk 2]
```diff
@@ -114,8 +114,8 @@
...
-			M << "<span class='revennotice'><b>An alien voice resonates from all around...</b></span><i> [msg]</I>"
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/revenant/revenant_abilities.dm]: this one is fucked up and honestly should have the `</span>` after the `</I>`

Comment (Cheridan) [inline code/game/gamemodes/miniantags/revenant/revenant_abilities.dm]: The msg bit is its own part. I can change it to use a different span but there wasn't a revenant span with italics.

File: code/game/gamemodes/miniantags/revenant/revenant_abilities.dm (hunk 1 above)
Comment (Cheridan) [inline code/game/gamemodes/miniantags/revenant/revenant_abilities.dm]: mm i see now

**Comment by Cheridan** [comment]
fixed

**Pull request merged**